src/deterministic.js

```diff
@@ -26,7 +26,7 @@ Module['thisProgram'] = 'thisProgram'; // for consistency between different buil
 
 function hashMemory(id) {
   var ret = 0;
-  var len = _sbrk();
```

Comment:
Are these changes needed because of strict mode?

Reply:
They are needed because closure now knows the actual arity of the native functions.   I could revert that part and re-introduce the closure annotation?   But I think I actually prefer closure warnings when we calling with the wrong number of args.. and the number of places where we save a couple of bytes by not passing these extra args was very low.